api/HTMLCanvasElement.json

```
@@ -736,7 +736,7 @@
               "version_added": "11"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "12"
```

Hi @vh13294 and welcome to BCD! According to this page https://webkit.org/blog/7956/new-webkit-features-in-safari-11/ (which I found from the other PR you linked to), this appeared on iOS 11. Once we update that, we can merge this PR. Thank you!

```suggestion
              "version_added": "11"
```